app/javascript/app/components/modal-png-download/modal-png-download-component.jsx

```diff
@@ -25,7 +23,17 @@ const modalPngDownloadComponent = ({
       header={<ModalHeader title={header} />}
     >
       <div ref={modalContentRef}>
-        <Icon className={styles.logo} icon={cwLogo} />
+        <svg
```

Comment:
We cant use the Icon component because we have to add a class? What about importing it from the assets folder and adding a class afterward?

Reply:
No, the reason is that through the `svg-sprite-loader` we are getting the logo from a sprite and the way it adds the `svg` to the DOM is through a [`#shadow-root`](https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot) so the module that we are using to create the `png` can't access the `svg path` inside it

Reply:
Ok, could you add this as a comment? Feel free to merge afterward

Reply:
On the code you mean?

Reply:
Yes, this is not so obvious at plain sight